dmriprep/utils/images.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+import nibabel as nb
+from nipype.utils.filemanip import fname_presuffix
+
+
+def extract_b0(in_file, b0_ixs, newpath=None):
```

Comment:
@josephmje could you open a new PR fixing these `newpath` to be consistent to the `out_path` of `average_images` below? I think that behavior is better.

Reply:
I agree. Will do